src/serac/coefficients/coefficient_extensions.hpp

```diff
@@ -85,16 +229,52 @@ mfem::Array<int> MakeAttributeList(
  * attribute 2.
  * @return An array holding the attributes that correspond to each element
  */
-mfem::Array<int> MakeBdrAttributeList(
-    mfem::Mesh& m, mfem::Coefficient& c,
-    std::function<int(double)> digitize = [](double v) { return v == 1. ? 2 : 1; });
+template <typename T>
+T MakeBdrAttributeList(mfem::Mesh& m, mfem::Coefficient& c, std::function<int(double)> digitize = digitize::equals1)
```

Comment:
Previously this method returned a `mfem::Array`. In this case I'm just allowing `std::vector` to be returned instead so it can be used as input to `Set` which lives outside of the mfem ecosystem.

I think the discussion to make this work better outside of the MFEM ecosystem is a slightly separate enhancement? This is more of an incremental move using the MFEM ecosystem as reference.